src/sourcerer/presentation/utils.py

```
@@ -57,13 +58,16 @@ def get_provider_service_by_access_credentials(
     )
 
     if not credentials_service:
-        return
+        return None
 
     provider_service_class = storage_provider_registry.get_by_provider(
         credentials.provider
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)